ct.sh

```diff
@@ -55,10 +55,10 @@ install_chart_testing() {
         exit 1
     fi
 
-    local arch
-    arch=$(uname -m)
-
+    local arch=$(uname -m)
```

Comment:
Keep this as it was to make Shellcheck happy.

```$ shellcheck ct.sh

In ct.sh line 58:
    local arch=$(uname -m)
          ^--^ SC2155: Declare and assign separately to avoid masking return values.

For more information:
  https://www.shellcheck.net/wiki/SC2155 -- Declare and assign separately to ...
```

Reply:
I'll get that fixed and add a PR to add the shellcheck action